docs/dev_install.rst

```diff
@@ -15,14 +15,32 @@ Installing Depedencies
 ----------------------
 
 - Make sure you are using Python 3.6/3.7/3.8
+- Make sure you have `CMake <https://cmake.org/>`_ installed and configured.
 - Execute
 
   .. code-block:: bash
 
     python3 -m pip install --user setuptools astpretty astor pytest opencv-python pybind11
     python3 -m pip install --user Pillow numpy scipy GitPython yapf colorama psutil autograd
 
-* (If on Ubuntu) Execute ``sudo apt install libtinfo-dev clang-8``. (``clang-7`` should work as well).
+* (**If on Ubuntu**) Execute ``sudo apt install libtinfo-dev clang-8``. (``clang-7`` should work as well).
+
+* (**If on macOS**) You could rely on `Homebrew <https://brew.sh/>`_ to install LLVM with clang for you.
```

Comment:
Thanks for adding an alternative way to config llvm on OS X! Our community follow *single-responsibility PR* style, so let's do a cherry-pick of the commits and focus this PR on CLI :-)

A quick question out of curiosity for now: when building LLVM from scratch, we add extra CMake arguments
```
cmake .. -DLLVM_ENABLE_RTTI:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86;NVPTX" -DLLVM_ENABLE_ASSERTIONS=ON
```

Does the version you get from homebrew work out-of-box? I thought that we need a customized LLVM.

Reply:
Thanks for the catch. Actually it was smooth for me to run the examples with Homebrew's llvm@8, but there's a segmentfault bitting me now with `ti test`. (The mismatches among the buildaguments at [here](https://github.com/Homebrew/homebrew-core/blob/ca64172118d2ba5e870f4b59a1b4e9e8932c9d9b/Formula/llvm%408.rb#L108) must have caused this :( )

I guess we do need a customized LLVM, I will remove this section.